nsxiv.desktop

```
@@ -3,6 +3,6 @@ Type=Application
 Name=nsxiv
 GenericName=Image Viewer
 Exec=nsxiv %F
-MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/png;image/tiff;image/x-bmp;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-tga;image/x-xpixmap;image/webp;
+MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/png;image/tiff;image/x-bmp;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-tga;image/x-xpixmap;image/webp;image/heic;
```

@XPhyro @eylles Is it possible to "glob" on desktop file? 

```suggestion
MimeType=image/*;
```

---

Don't see anything about globbing on https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html, so I guess not.

---

There were lots of other loaders added recently as well. Including svg, heif, avif, jxl etc. Not a fan of having to manually add these on each imlib2 release.

Not to mention, there's also a couple 3rd party imlib2 loaders and users can disable any loader at imlib2 build time.

---

Well, the only other thing we can do about it on our end is to preemptively add `image/X` entries since globbing is not supported. Perhaps we add this to the readme or to #242 and tell the user to update the desktop entry depending on their imlib2?

In any case, I think it makes sense to keep the desktop entry in-line with the default configuration of imlib2.

---

I'm thinking maybe grabbing a list of all image mimetypes and adding them in. If it ends up opening a non-supported image, it's a non-fatal problem. nsxiv will just reject it.

---

`nsxiv` will reject it, but this will prevent the image from being opened with another application that actually supports it (assuming `xdg-open` or similar workflow).

---

Yeah, similar problem will occur even if we keep our desktop file in sync with imlib2's default config. For example if someone's on a distro with older imlib2 version where svg isn't supported.